BooruSharp/Booru/UrlFormat.cs

```diff
@@ -1,10 +1,25 @@
 ﻿namespace BooruSharp.Booru
 {
+    /// <summary>
+    /// Indicates the API URL format that will be used to create API requests.
+    /// </summary>
     public enum UrlFormat
     {
-        postIndexJson, //post/index.json
-        indexPhp, // index.php?page=dapi&s=post&q=index&json=1
-        danbooru, // posts.json
-        sankaku // posts
+        /// <summary>
+        /// <c>post/index.json</c>
+        /// </summary>
+        postIndexJson,
+        /// <summary>
+        /// <c>index.php?page=dapi&amp;s=post&amp;q=index&amp;json=1</c>
+        /// </summary>
+        indexPhp,
+        /// <summary>
+        /// <c>posts.json</c>
+        /// </summary>
+        danbooru,
+        /// <summary>
+        /// <c>posts</c>
+        /// </summary>
+        sankaku
```

Comment:
I couldn't come up with anything more fancy.

Reply:
Maybe you can write [url]/posts or something so it's more explicit for danbooru and sankaku?